cardano-client-demo/cardano-client-demo.cabal

```diff
@@ -0,0 +1,26 @@
+cabal-version:       2.4
+name:                cardano-client-demo
+version:             0.1.0.0
+synopsis:            A simple demo cardano-node client application
+description:         This is intended to demonstrate how to write simple
+                     applications that interact with the cardano-node,
+                     using the cardano-api.
+license:             Apache-2.0
+license-file:        LICENSE
+author:              IOHK
+maintainer:          operations@iohk.io
+license-files:
+  LICENSE
+  NOTICE
+extra-source-files:  CHANGELOG.md
+
+executable cardano-client-demo
+  main-is:             Main.hs
+  other-modules:
+  build-depends:       base,
+                       cardano-api,
+                       cardano-ledger,
```

Comment:
```suggestion
```
We should not need to import `cardano-ledger` directly. (That's actually the old Byron ledger)